src/rules/officer-rules.ts

```diff
@@ -0,0 +1,35 @@
+// Officer Rules
+export const firstNameRules: Array<Function> = [
+  v => !!v || 'A first name is required',
+  v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+  v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+  v => (v?.length <= 30) || 'Cannot exceed 30 characters' // maximum character count
+]
+
+export const middleNameRules: Array<Function> = [
+  v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+  v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+  v => (!v || v.length <= 30) || 'Cannot exceed 30 characters' // maximum character count
+]
+
+export const lastNameRules: Array<Function> = [
+  v => !!v || 'A last name is required',
+  v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+  v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+  v => (v?.length <= 30) || 'Cannot exceed 30 characters' // maximum character count
+]
+
+export const orgNameRules: Array<Function> = [
```

Comment:
How about moving this to a new file, and also emailRules below?

Reply:
This is a new file :) 
I put them here as they are rules on properties associated with our definition of the `officer`
I don't mind re organizing this, just not sure why for email or org vs the name rules?

Reply:
It just seems that orgNameRules and emailRules don't belong in a file named officer-rules ...?

Reply:
Yea, the email rule could be pulled out but in the context of our apps an organization is a `PartyType `. A property defined on `officers` only, i think*. 
Yea, it doesn't seem totally intuitive, i see what you are saying for sure.

Reply:
I had an epiphany this morning... it's revolutionary. 
Will extract these rules out into two files, one called `name-rules` and the other called `contact-rules` 😁 
Updating!

Reply:
Vive this kind of revolution!
(Or could just call it `rules.ts`, then you only need to import 1 file.)

Reply:
I just looked at your changes re: this. They look good too, so I'm closing this conversation.